api/reviewer_api/services/documentservice.py

```diff
@@ -85,13 +134,17 @@ def __filterrecords(self, records):
             if record["recordid"] is None:
                 attchments.append(record)
         return parentrecords, parentswithattachments, attchments
-       
+
     def __getpagecountandfilename(self, record, properties):
         pagecount = 0
         filename = record["filename"] if "filename" in record else None
         documentid = None
+        version = 0
```

Comment:
to fix "4218UnboundLocalError: local variable 'version' referenced before assignment" error